tests/node_converters/lrn_test.py

```diff
@@ -0,0 +1,36 @@
+from random import randrange
```

Comment:
https://github.com/ENOT-AutoDL/onnx2torch/runs/8241926516?check_suite_focus=true

Reply:
To check your code before you committing it you can use [precommit hook](https://pre-commit.com/), we already set it up, just type `pre-commit install` in the root of onnx2torch directory.
This hook will check your changes when your type `git commit` and discard the commit if there is a problem.

Reply:
I installed pre-commit and isort and got the imports sorted! In order for pre-commit checks to pass, I had to rename my default branch on my fork (as it wouldn't allow pushing to branches called "main"--totally understand why but it was giving me issues on my fork!). Accordingly, I think that's caused this PR thread to die so I will try to reopen the PR using my renamed branch ... and then hopefully we'll be all set. Thank you for your patience!

Reply:
It is not good practice even for forked repos:
1. there will be sync conflicts between main repository and your fork with commits on the top of main
2. to check new PR reviewer makes something like `git fetch origin pull/96/head:main ` that efficiently rewrites his main :)

Also, pre-commit only works on your side, after checking code you can disable it for pushing (`pre-commit uninstall`)

Anyway everything is fine :+1:

Reply:
Oooh! I wish GitHub would choose a different default branch name for forked repos then. Now I know for next time!